model/SearchEngine/Driver/Elasticsearch/ElasticSearch.php

```diff
@@ -37,7 +38,7 @@
 use oat\taoAdvancedSearch\model\SearchEngine\Service\IndexPrefixer;
 use Psr\Log\LoggerInterface;
 
-class ElasticSearch implements SearchInterface, TaoSearchInterface
+class ElasticSearch implements SearchInterface, TaoSearchInterface, BoolSearchInterface
```

Comment:
why not to add` boolQuery()` to `TaoSearchInterface`  or `SearchInterface` ?

Reply:
To not break existing interface

Reply:
We do not need this interface or a new method, all is already supported by the ElasticSearch implementation using simple query https://github.com/oat-sa/extension-tao-advanced-search/pull/140/files#r1937030160